medusa/common.py

```diff
@@ -179,21 +181,21 @@ class Quality(object):
         None: "None",
         NONE: "N/A",
         UNKNOWN: "Unknown",
-        SDTV: "HDTV",
+        SDTV: "",
```

Comment:
yes it has. logging and everything else

whenever you call `Quality.sceneQualityStrings[ep_qual]` and ep_qual is SDTV, it will show a empty string